### PR TITLE
Fixed component name and removed trailing commas in readme file.

### PR DESCRIPTION
**Pull request by Mohammed9531**
@rychkog Can you please review and merge this PR?

Please let me know if you have any questions.

### CODE REVIEW

**Comment by rychkog** [comment]
@Mohammed9531 LGTM :) Didn't have time to review at PRs during the week.
Will have a look at another your RP

File: README.md [hunk 1]
```diff
@@ -40,9 +40,9 @@ import { BrowserModule } from '@angular/platform-browser';
 import { TreeModule } from 'ng2-tree';
 
 @NgModule({
-  declarations: [MyComponent],
+  declarations: [AppComponent],
```
Comment (rychkog) [inline README.md]: `MyComponent` is correct, cause I was describing the one that is below module definition at https://github.com/valor-software/ng2-tree#clapper-usage. It is not connected to the demo

Comment (Mohammed9531) [inline README.md]: Ah! I didn't realized that. I changed it cause the import above was for AppComponent. I will fix that.

Comment (Mohammed9531) [inline README.md]: Ah! I didn't realize that. I changed it cause the import above was for AppComponent. I will fix that.

**Comment by rychkog** [comment]
@Mohammed9531 Closing this one as well because changes from it are in `develop`

**Pull request closed without merging**